pkg/apis/serving/v1beta1/revision_validation.go

```diff
@@ -111,7 +111,13 @@ func (current *RevisionTemplateSpec) VerifyNameChange(ctx context.Context, og Re
 
 // Validate implements apis.Validatable
 func (rs *RevisionSpec) Validate(ctx context.Context) *apis.FieldError {
-	errs := rs.ContainerConcurrency.Validate(ctx).ViaField("containerConcurrency")
+	var errs *apis.FieldError
+	if rs.ContainerConcurrency != nil {
```

Comment:
Why drop the `ViaField("containerConcurrency")`?

I might pull this out into a `ValidateContainerConcurency` function.

Reply:
Yeah that was unnecessary 😅 I've updated the PR. Would you please review again?